index.js

```diff
@@ -119,4 +142,10 @@ console.log('hello world')
  * Example Output: 2, 10, 20
  */
 
-// TODO: ADD CODE HERE
+function multiplyByTwo(number){
+    return number * 2;
+}
+    const num1 = multiplyByTwo(1);
+    const num2 = multiplyByTwo(5);
+    const num3 = multiplyByTwo(10);
+    console.log(`${num1},${num2},${num3}`);
```

Comment:
nice use of string literal!
could include a space between the numbers here like

`console.log(`${num1}, ${num2}, ${num3}`);`